.docker/run.sh

```diff
@@ -0,0 +1,35 @@
+#! /bin/bash
+
+# name of image
+IMAGE=lab-website-renderer:latest
+
+# name of running container
+CONTAINER=lab-website-renderer
+
+# choose platform
+PLATFORM=""
+
+# default docker run command
+DOCKER_RUN="docker run"
+
+# fix interactive on windows
+if [[ $OSTYPE == msys* ]]; then
+    DOCKER_RUN="winpty docker run"
+fi
+
+# build docker image
+docker build ${PLATFORM} \
+    --tag ${IMAGE} \
+    --file ./.docker/Dockerfile .
```

Comment:
As mentioned in a previous comment, I'd suggest you resolve and pass the BUNDLER build arg here, i.e.:
```suggestion
# infer the bundler version from the Gemfile
BUNDLER=$( grep -A 1 'BUNDLED WITH' Gemfile.lock | tail -n 1 | xargs )

# build docker image
docker build ${PLATFORM} \
    --tag ${IMAGE} \
    --build-arg BUNDLER=${BUNDLER} \
    --file ./.docker/Dockerfile .
```